test/integration/lro.spec.ts

```diff
@@ -540,7 +540,7 @@ describe("Custom Headers", () => {
     await client.lROsCustomHeader.beginPost202Retry200AndWait(options);
     check200(lastResponse);
   });
-});
+}).timeout(5000);
```

Comment:
Is there a way we can avoid doing this long timeout?

I think we can use Sinon fakeTimers: https://sinonjs.org/releases/v1.17.7/fake-timers/

Reply:
The ci somehow fails and report some errors like timeout if I don't add this. I am also quite superised about this as this should not be related to my changes.

Reply:
removed.